src/k8s-extension/azext_k8s_extension/partner_extensions/BackupAgent.py

```diff
@@ -0,0 +1,68 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=unused-argument
+
+import datetime
+import json
+
+from ..utils import get_cluster_rp_api_version
+
+from knack.log import get_logger
+
+from azure.cli.core.azclierror import InvalidArgumentValueError
+from azure.cli.core.commands.client_factory import get_subscription_id
+from msrestazure.azure_exceptions import CloudError
+
+from ..vendored_sdks.models import Extension
+
+from .DefaultExtension import DefaultExtension
+
+from .._client_factory import (
+    cf_resources)
+
+logger = get_logger(__name__)
+
+
+class AzureBackupAgent(DefaultExtension):
+    def __init__(self):
+        # constants for configuration settings.
+        self.DEFAULT_RELEASE_NAMESPACE = 'backupagent-ns'
```

Comment:
This default release namespace is part of the extension registration so it is not required to be statically coded into the CLI